### DISCUSSION TITLE
I want a sandbox with no security isolation

### DISCUSSION THREAD

**Discussion by lmou523** [sandboxie-plus/Sandboxie#2986] [Q&A Feedback]
I want a sandbox that doesn't handle security isolation, tokens, files, registry, network, desktop, window class names, and can be dragged and dropped.
This sandbox does not perform isolation, only virtualization.Is that possible?
Existing sandboxes without security isolation are poorly done and full of bugs.
![image](https://github.com/sandboxie-plus/Sandboxie/assets/59944420/07905fdc-c674-4549-840b-10e3f400a6a4)
Even if I just use NoSecurityIsolation=y, explorer.exe will get an error, and many features will fail, such as double-clicking to open the xlsx document

**Comment by lmou523** [comment]
My ultimate goal is to find out what functionality is not needed if you're just doing application virtualization
For example, use these configurations:
OriginalToken=y
UnrestrictedToken=y
NoSecurityIsolation=y
OpenCredentials=y
RunServicesAsSystem=y
NoSandboxieDesktop=y
NoSandboxieConsole=y
NoSysCallHooks=y
OpenWinClass=*

**Comment by lmou523** [comment]
I found that when I used the green box (without security isolation), the gui agent would not start, but at the same time many gui hooks needed it (dde? getclassname?) This leads to many problems

**Comment by DavidXanatos** [comment]
I'll look into this and debug it soon we need to improve the green boxes to be in all cases more compatible then yellow once and not just in some, as it is now some things work great in green much better then in yellow but others just fail completely.

**Reply by lmou523** [reply]
> I'll look into this and debug it soon we need to improve the green boxes to be in all cases more compatible then yellow once and not just in some, as it is now some things work great in green much better then in yellow but others just fail completely.

That's what I hope!
